app-localize-behavior.html

```diff
@@ -227,6 +232,12 @@
     };
   },
 
+  __languageChanged: function(newLang, oldLang) {
+    if (newLang === oldLang) return;
+    this.language = newLang;
+    this.fire('app-localize-behavior-language-changed', newLang);
```

Comment:
I would prefer the `language-changed`, since it's fairly unlikely this random event will bubble in the same place as this event (and if it does, you probably have bigger problems, such as this element having two `language` attributes that stomp over each other